### PR TITLE
Adds refresh audit data in crontasks list

### PR DESCRIPTION
**Pull request by fitn**
**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: install_pim/manual/crontab_tasks_ee.rst.inc [hunk 1]
```diff
@@ -38,6 +39,7 @@ In the following example, these lines will run the versioning calculation at 5am
 
     # m  h  dom  mon  dow  command
     0 5   *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console pim:versioning:refresh --env=prod > /path/to/installation/pim-enterprise-standard/var/logs/refresh_versioning.log 2>&1
+    1 *   *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console bin/console akeneo:connectivity-audit:update-data --env=prod > /path/to/installation/pim-enterprise-standard/var/logs/refresh_audit_data.log 2>&1
```
Comment (ahocquard) [inline install_pim/manual/crontab_tasks_ee.rst.inc]: The sentence 

`In the following example, these lines will run the versioning calculation at 5am, the rules calculation at 6pm and the aggregate volume at 10pm every day:`

is not up to date. It misses this job. Maybe it's not mandatory to add this line for every jobs btw (it's not done for quality insight jobs).

File: install_pim/manual/crontab_tasks_ee.rst.inc [hunk 2]
```diff
@@ -9,12 +9,13 @@ Edit your crontab with ``crontab -e`` and configure your tasks.
 
     Be sure to update the cron of the user used to install the PIM, to be able to run the command
 
-You have to add these 7 commands to your local installation:
+You have to add these 10 commands to your local installation:
```
Comment (ahocquard) [inline install_pim/manual/crontab_tasks_ee.rst.inc]: :+1:

File: install_pim/manual/crontab_tasks.rst.inc [hunk 3]
```diff
@@ -25,7 +26,8 @@ In the following example, these lines will run the versioning calculation at 5am
 
     # m  h  dom  mon  dow  command
     0 5  *    *    *    php /path/to/installation/pim-community-standard/bin/console pim:versioning:refresh --env=prod > /path/to/installation/pim-community-standard/var/logs/refresh_versioning.log 2>&1
-    0 22  *    *    *    php /path/to/installation/pim-community-standard/bin/console pim:volume:aggregate --env=prod > /path/to/installation/pim-community-standard/var/logs/volume_aggregate.log 2>&1
+    1 *  *    *    *    php /path/to/installation/pim-community-standard/bin/console akeneo:connectivity-audit:update-data --env=prod > /path/to/installation/pim-community-standard/var/logs/refresh_audit_data.log 2>&1
```
Comment (ahocquard) [inline install_pim/manual/crontab_tasks.rst.inc]: `In the following example, these lines will run the versioning calculation at 5am and the aggregate volume at 10pm every day:`

not up to date. (Same comment as below, maybe useless)

File: install_pim/manual/crontab_tasks_ee.rst.inc (hunk 1 above)
Comment (pierrejolly) [inline install_pim/manual/crontab_tasks_ee.rst.inc]: bin/console is writed twice

**Comment by ahocquard** [comment]
Here a link of a PR I did to remove things about the completeness if it can help you also to know all the places where the information is: https://github.com/akeneo/pim-docs/pull/1342/files

**Comment by fitn** [comment]
@ahocquard I added doc for the flexibility.
I am OK to fix the doc for the cron tasks if you need help on this topic. Just I don't want to this in this PR ;)

**Comment by ahocquard** [comment]
@fitn I will ask to fix the doc to the raccoons, don't change it in your PR.
If you read the file I linked to you, I think  you will see that the crontab is also duplicated for the flexibility env. But without your task: this is the missing information.

For the  other comments, don't hesitate to ask me directly in the thread if it's not clear.

File: install_pim/manual/crontab_tasks.rst.inc [hunk 4]
```diff
@@ -9,12 +9,13 @@ Edit your crontab with ``crontab -e`` and configure your tasks.
...
+You have to add these 3 commands to your local installation:
 
 .. code-block:: bash
     :linenos:
 
     php /path/to/installation/pim-community-standard/bin/console pim:versioning:refresh --env=prod        # processes pending versions
+    php /path/to/installation/pim-community-standard/bin/console akeneo:connectivity-audit:update-data --env=prod # refresh audit data for connections
     php /path/to/installation/pim-community-standard/bin/console pim:volume:aggregate --env=prod          # recalculates the number of product values
 
 In the following example, these lines will run the versioning calculation at 5am and the aggregate volume at 10pm every day:
```
Comment (fitn) [inline install_pim/manual/crontab_tasks.rst.inc]:
```suggestion
In the following example, these lines will run the versioning calculation at 5am, the refresh of audit data every hour (at 00:01) and the aggregate volume at 10pm every day:
```

File: cloud_edition/flexibility_mode/docs/crontasks.rst [hunk 5]
```diff
@@ -109,6 +111,8 @@ The default crontab at the moment on our Flexibility environments is the followi
 +=========================================================+===================+============================================+
 | pim:versioning:refresh --env=prod                       | 30 1 \* \* \*     | At 01:30 AM                                |
 +---------------------------------------------------------+-------------------+--------------------------------------------+
+| akeneo:connectivity-audit:update-data --env=prod        | 1 \* \* \* \*     | At 00:01 AM, every hour                    |
```
Comment (ahocquard) [inline cloud_edition/flexibility_mode/docs/crontasks.rst]:
```suggestion
| akeneo:connectivity-audit:update-data --env=prod        | 1 \* \* \* \*     | Every hour                    |
```

File: install_pim/manual/crontab_tasks.rst.inc [hunk 6]
```diff
@@ -9,23 +9,25 @@ Edit your crontab with ``crontab -e`` and configure your tasks.
...
 
 .. code-block:: bash
     :linenos:
 
     php /path/to/installation/pim-community-standard/bin/console pim:versioning:refresh --env=prod        # processes pending versions
+    php /path/to/installation/pim-community-standard/bin/console akeneo:connectivity-audit:update-data --env=prod # refresh audit data for connections
     php /path/to/installation/pim-community-standard/bin/console pim:volume:aggregate --env=prod          # recalculates the number of product values
 
-In the following example, these lines will run the versioning calculation at 5am and the aggregate volume at 10pm every day:
+In the following example, these lines will run the versioning calculation at 5am, the refresh of audit data every hour (at 00:01) and the aggregate volume at 10pm every day:
```
Comment (ahocquard) [inline install_pim/manual/crontab_tasks.rst.inc]:
```suggestion
In the following example, these lines will run the versioning calculation at 5am, the refresh of audit data every hour (every hour) and the aggregate volume at 10pm every day:
```

File: install_pim/manual/crontab_tasks_ee.rst.inc [hunk 7]
```diff
@@ -38,6 +39,7 @@ In the following example, these lines will run the versioning calculation at 5am
 
     # m  h  dom  mon  dow  command
     0 5   *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console pim:versioning:refresh --env=prod > /path/to/installation/pim-enterprise-standard/var/logs/refresh_versioning.log 2>&1
+    1 *   *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console akeneo:connectivity-audit:update-data --env=prod > /path/to/installation/pim-enterprise-standard/var/logs/refresh_audit_data.log 2>&1
```
Comment (ahocquard) [inline install_pim/manual/crontab_tasks_ee.rst.inc]: missing update of the sentence as in CE I think

**Pull request merged**